src/main/resources/feedbackQuestionMcqEditFormOptionFragment.html

```diff
@@ -1,19 +1,17 @@
-<div class="margin-bottom-7px" id="mcqOptionRow-${i}-${questionNumber}">
-  <div class="input-group width-100-pc">
-    <span class="input-group-addon">
-      <input type="radio" class="disabled_radio" disabled>
-    </span>
-    <input class="form-control" type="text" disabled
-        name="${Const.ParamsNames.FEEDBACK_QUESTION_MCQCHOICE}-${i}"
-        id="${Const.ParamsNames.FEEDBACK_QUESTION_MCQCHOICE}-${i}-${questionNumber}"
-        class="mcqOptionTextBox" value="${mcqChoiceValue}">
-    <span class="input-group-btn">
-      <button class="btn btn-default removeOptionLink" type="button" id="mcqRemoveOptionLink"
-          onclick="removeMcqOption(${i},${questionNumber})"
-          style="display:none" tabindex="-1">
-        <span class="glyphicon glyphicon-remove">
-        </span>
-      </button>
-    </span>
-  </div>
+<div class="input-group margin-bottom-7px mcqChoiceRow-${i}-${questionNumber}" id="mcqOptionRow-${i}-${questionNumber}">
```

Comment:
Does the `id` here serve any purpose anymore?

Reply:
Yes, As I am now using `id` instead of class to remove options and weights in `removeMcqOptions`, so I am using this ID to get to the option row. I have removed the class though.